src/processing/resources/index.js

```diff
@@ -5,12 +5,19 @@ import scriptProcessor from './script';
 import stylesheetProcessor from './stylesheet';
 import * as urlUtil from '../../utils/url';
 import { encodeContent, decodeContent } from '../encoding';
+import { platform } from 'os';
+
+const IS_WIN32 = platform() === 'win32';
+const DISK_RE  = /^[A-Za-z]:/;
 
 function getResourceUrlReplacer (ctx) {
     return function (resourceUrl, resourceType, charsetAttrValue, baseUrl) {
         if (!urlUtil.isSupportedProtocol(resourceUrl) && !urlUtil.isSpecialPage(resourceUrl))
             return resourceUrl;
 
+        if (IS_WIN32 && ctx.dest.protocol === 'file:' && DISK_RE.test(resourceUrl))
```

Comment:
`/usr/home/index.html` is already relative path which can be resolved via the `url.resolve` function